Mathlib/LinearAlgebra/Basis/Defs.lean

```diff
@@ -69,15 +69,12 @@ open Function Set Submodule Finsupp
 variable {ι : Type*} {ι' : Type*} {R : Type*} {R₂ : Type*} {K : Type*}
 variable {M : Type*} {M' M'' : Type*} {V : Type u} {V' : Type*}
 
-section Module
+namespace Module
```

Comment:
Pre-existing: I suspect almost all of this file should be inside the `Basis` namespace.
(But that's out of scope for this PR.)